tasks/objects/objects.js

```diff
@@ -2,21 +2,33 @@
  * create object person with firstName and lastName properties
  * implement method getFullName that returns full name
  */
-const person = //put you object here
+let person = {
+    firstName: 'firstName',
+    lastName: 'lastName',
+    getFullName: function () {
```

Comment:
you can use the short way to define the methods:
getFullName() {
  return this.firstName + ' ' + this.lastName;
}